deployment/articles-dxp/01-saml/01-intro.markdown

```diff
@@ -0,0 +1,271 @@
+---
+header-id: authenticating-using-saml
+---
+
+# Authenticating Using SAML
+
+The SAML (Security Assertion Markup Language) adapter provides Single Sign On
+(SSO) and Single Log Off (SLO) in your deployment. Each @product@ instance
+serves as either the Service Provider (SP) or the Identity Provider (IdP). An
+identity provider is a trusted provider that provides single sign-on for users
+to access other websites. A service provider is a website that hosts
+applications and grants access only to identified users with proper credentials.
+
+| **Note:** A single @product@ instance is *either* the SP or the IdP in your SSO
+| setup; it can't be both. You can, however, use separate instances for both
+| purposes (for example, one instance is the SP and another is the IdP).
+
+Below is background on how SAML works. To jump right to its configuration, see
+the articles on 
+[Setting Up SAML as an Identity Provider](/docs/7-2/deploy/-/knowledge_base/deploy/setting-up-liferay-as-a-saml-identity-provider)
+or 
+[Setting Up SAML as a Service Provider](/docs/7-2/deploy/-/knowledge_base/deploy/setting-up-liferay-as-a-saml-service-provider) 
+for instructions on using the 
+[SAML adapter](https://web.liferay.com/marketplace/-/mp/application/15188711). Use
+the instructions to make the conceptual magic from this article come to life!
+
+| **Note:** If you're migrating from a Liferay SAML adapter prior to version
+| 3.1.0, your properties are automatically migrated to settings. Please see the
+| [Configuring SAML](/discover/deployment/-/knowledge_base/7-1/configuring-saml)
+| article for details on settings.
+
+## Important SAML URLs
+
+For reference, here are a few important SAML URLs.
+
+This URL is the default location of the metadata XML file:
+
+    [host]:[port]/c/portal/saml/metadata
+
+Note that when configuring SAML, no importing of SAML certificates is required.
+@product@ reads certificates from the SAML metadata XML file. If you want a
+third-party application like Salesforce to read a Liferay SAML certificate, you
+can export the @product@ certificate from the keystore. The default keystore
+file is `[Liferay Home]/data/keystore.jks`. 
```

Comment:
Add: It can be changed through System Settings > SSO > SAML Configuration > "Key Store Path" property.

Reply:
Done!